src/convertToObject.js

```diff
@@ -6,7 +6,57 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString
```

Comment:
The variable name `splittedString` is not ideal. Consider renaming it to `splitString` for better readability.